crowbar_framework/app/models/service_object.rb

```diff
@@ -1125,6 +1125,12 @@ def apply_role(role, inst, in_queue)
 
       save_it = false
 
+      # reset possible previous error flags at the node
```

Comment:
So, this is the place where we iterate over nodes during apply_role, so I thought it might be a good place to reset the error flag.

However, what if the apply_role after the one which failed is working with different set of nodes? That we would have some nodes where the error message is still saved, right? After all, it might look that reseting the flag _after_ the apply role (like in previous commits, still present in this PR) might be better idea...